.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+<!--
+This form is for bug reports and feature requests ONLY!
+
+For high severity issues, please contact AWS Support directly.
+
+Please provide the following information:
+-->
+
+<!--
+BUG REPORT or FEATURE REQUEST ?
+
+Uncomment only one of the below items
+-->
+
+<!-- ### Bug Report -->
+<!-- ### Feature Request -->
+
+
+### Summary
+<!-- Please provide a brief outline of the bug or feature request -->
+
+
+### Description
+<!-- Provide detailed information about the issue -->
+
+
+<!-- -->
+### Expected Behavior
+
+
+### Supporting Log Snippets
+<!-- Please provide supporting log information for this issue -->
+
+
+### Stacktrace
+
+
+### Environment Details
+
+
+### Additional Information
+<!--
+Please have a look at https://github.com/awslabs/ecs-logs-collector
```

Comment:
Isn't this the same as the log section above?  Also, it'd be good to actually enumerate the details we want.